demo/admin/codegen.ts

```diff
@@ -1,9 +1,10 @@
+import { resolve } from "node:path";
+
 import { CodegenConfig } from "@graphql-codegen/cli";
 import { readFileSync } from "fs";
 import { buildSchema } from "graphql";
 
-const schema = buildSchema(readFileSync("./schema.gql").toString());
-
+const schema = buildSchema(readFileSync(resolve(__dirname, "./schema.gql")).toString());
```

Comment:
Why is this necessary? 🤔

Reply:
knip is somehow loading all source files and tries to resolve schema.gql. due to knip run's in the root directory it can not find schema.gql file.

Reply:
Because we have no monorepo in the starter. In the starter we are executing 3 times knip - in each service (api/admin/site) so the execution directory is different like in this repository.